File: blinkled/blinkled.py
#!/usr/bin/env python

# Blink an LED on Pin 16 to GND
# tested on ODROID C4

import odroid_wiringpi as wpi
import time

wpi.wiringPiSetup()
# pin 16 ist output
wpi.pinMode(4,1)

while True:
	wpi.digitalWrite(4,1)
	time.sleep(1)
	wpi.digitalWrite(4,0)
	time.sleep(1)

File: blinkled/blinkled.c
// Example for blinking a LED on pin 16 to GND
// on Odroid C4

#include <wiringPi.h>

int main(void) {
	puts("led an pin 16");
	wiringPiSetup();
	pinMode(4, OUTPUT);
	
	for(;;)	{
		digitalWrite(4,HIGH);
		delay(1000);
		digitalWrite(4,LOW);
		delay(1000);
	}
	return(0);
}

